### PR TITLE
Fixed lumen support

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: src/AwsServiceProvider.php [hunk 1]
```diff
@@ -26,8 +28,10 @@ public function boot()
     {
         $source = realpath(__DIR__ . '/../config/aws.php');
 
-        if (class_exists('Illuminate\Foundation\Application', false)) {
+        if ($this->app instanceof LaravelApplication && $app->runningInConsole()) {
             $this->publishes([$source => config_path('aws.php')]);
+        } elseif ($this->app instanceof LumenApplication) {
+            $app->configure('aws');
```
Comment (jeskew) [inline src/AwsServiceProvider.php]: Should this be `$this->app`?

Comment (GrahamCampbell) [inline src/AwsServiceProvider.php]: yes, oops

**Comment by jeskew** [comment]
I think I'd like to add a Lumen version of the ServiceProvider test.

**Comment by GrahamCampbell** [comment]
> I think I'd like to add a Lumen version of the ServiceProvider test.

That won't be possible since you can't install laravel and lumen at the same time.

**Pull request merged**